plot_features.py

```diff
@@ -97,29 +101,35 @@ def divide_poly_segments(points): #{{{
 
     return outPoints #}}}
 
-def plot_poly(mapInfo, points, color, filled=True): #{{{
-
-    points = divide_poly_segments(points)
+def plot_feature(mapInfo, geomtype, geom, **kwargs): #{{{
 
     refProjection = cartopy.crs.PlateCarree()
 
-    for mapType in mapInfo:
-        (ax, projection, plotFileName, fig) = mapInfo[mapType]
-        x = points[:,0]
-        y = points[:,1]
-        ax.fill(x, y, transform=refProjection, color=color, alpha=0.4, linewidth=2.0, zorder=3)
-
-    return #}}}
-
-def plot_point(mapInfo, points, marker, color): #{{{
+    genprop = {'transform':refProjection, 'alpha':0.4, 'linewidth':2.0}
 
-    points = np.asarray(points)
+    # merge plot properties
+    if kwargs is not None:
+        plotprop = genprop.copy()
```

Comment:
I think if `kwargs` is `None`, `plotprop` never gets assigned.  Since you're not reusing genprop for anything, and you just rename that to `plotprop` and not do the copy operation here?